Text skewed after Max IOPS per share

CHDAFNI-MSFT opened the pull request:
Looks like 'Storage' gets skewed and no longer follows format after 'Max IOPS per share'. Also... how do you edit these documents in a friendly way outside of the Git raw text editor. Apologies else I'd give the edit a try

azurecla commented:
Hi __@eerhardt__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

eerhardt commented:
I needed to fix the code in the other 2 places it was copied - Queues and Tables.  We will need all 3 commits for this issue to be fixed.

azurecla commented:
Hi __@eerhardt__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

tysonn commented:
Contributor is a MSFT employee.

CHDAFNI-MSFT committed:
Text skewed after Max IOPS per share

Looks like 'Storage' gets skewed and no longer follows format after 'Max IOPS per share'. Also... how do you edit these documents in a friendly way outside of the Git raw text editor.
